torch/onnx/_internal/fx/onnxfunction_dispatcher.py

```diff
@@ -430,7 +446,7 @@ class _OnnxSchemaChecker:
     provides a matching score to indicate how well the OpSchema matches the input and
     kwargs types.
 
-    There are three types of ONNX overloads in torchlib:
+    There are two types of ONNX overloads in torchlib:
```

Comment:
```suggestion
    There are two types of ONNX variants in torchlib:
```
does this change make sense? If so we should adopt this convention to separate concept from aten overload.